src/ray/common/grpc_util.h

```diff
@@ -38,23 +39,21 @@ class MessageWrapper {
   /// The input message will be **copied** into this object.
   ///
   /// \param message The protobuf message.
-  explicit MessageWrapper(const Message &message)
```

Comment:
Checkout modern effective C++ item41, passing by value is the best way to 
- handle both lvalue and rvalue overload
- with ownership held